### PR TITLE
feature/export-mock-responses

### PR DESCRIPTION
**Pull request by zjaries**
- export mock responses from index

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,16 +12,13 @@
...
   "exports": {
-    ".": {
-      "require": "./dist/index.cjs",
-      "import": "./dist/index.js"
-    },
-    "./types": "./dist/index.d.ts",
-    "./mockResponses/*": "./dist/mockResponses/*"
+    "require": "./dist/index.cjs",
+    "import": "./dist/index.js",
+    "types": "./dist/index.d.ts"
```
Comment (zjaries) [inline package.json]: I was really trying to get the mock responses to export from `tincheck/mock-responses`

I had it working as far as being able to import the files running in our dev. But eslint was really struggling to resolve the types without updating the main repo module resolution

Comment (brandon-pereira) [reply package.json]: The issue with this approach (I tried this) is it bundles all the files in dist/index.js so the bundle size increases drastically. What you could try is running `tsup` to compile mockResponses.ts file in a sequential build step

Comment (zjaries) [reply package.json]: I was doing that
I'll maybe push what I had, if you like it we can publish

Maybe if it's through npm the resolution will work. With linking it wouldn't

File: package.json [hunk 2]
```diff
@@ -14,14 +14,18 @@
...
+      "import": "./dist/index.js",
+      "types": "./dist/index.d.ts"
     },
-    "./types": "./dist/index.d.ts",
-    "./mockResponses/*": "./dist/mockResponses/*"
+    "./mock-responses": {
+      "require": "./dist/mockResponses.cjs",
+      "import": "./dist/mockResponses.js",
+      "types": "./dist/mockResponses.d.ts"
+    }
```
Comment (zjaries) [inline package.json:24]: This is what I was trying to do

**Pull request closed without merging**